### PR TITLE
apl: disable werror

### PR DESCRIPTION
**Pull request by oreo639**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/apl/template [hunk 1]
```diff
@@ -13,7 +13,7 @@ homepage="http://www.gnu.org/software/apl"
 distfiles="${GNU_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
 checksum=144f4c858a0d430ce8f28be90a35920dd8e0951e56976cb80b55053fa0d8bbcb
 nocross="execs target binary: ./Gtk_server --ev1 > Gtk_events1.asciidoc"
-CXXFLAGS="-Wno-format-truncation -Wno-class-memaccess -Wno-maybe-uninitialized -Wno-restrict -Wno-cpp"
+export CXX_WERROR='no'
```
Comment (sgn) [inline srcpkgs/apl/template:16]: I think this should go into `configure_args`.

**Comment by sgn** [comment]
The build seems to do some sneaky with `CFLAGS`, `-g` has been eaten.

**Comment by oreo639** [comment]
> The build seems to do some sneaky with `CFLAGS`, `-g` has been eaten.

Hrm. It does show up when compiling with debug packages enabled, is that expected?

**Comment by sgn** [comment]
We need to patch their config to not screw with our flags.

**Pull request merged**